src/Components/Dialogs/GetPerson/getPerson.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+import { Path } from '../../../core/constants';
+
+import style from './getPerson.module.css';
+
+const GetPerson = (props) => {
+  const { items } = props;
+  const itemsArray = Object.values(items);
+
+  const sd = ({ id, name }) => (<div key={id}><NavLink to={`${Path.dialog}/${id}`} className={style.item}>{name}</NavLink></div>);
```

Comment:
Consider renaming "Path" to "ROUTES" and "dialog" to "DIALOG" - to actually indicate that they are constants.

Reply:
Okey I changed